Spec/README.md

```diff
@@ -501,11 +501,26 @@ Each file/dir has the following purpose:
 
 **[CONTRIBUTORS.txt](https://github.com/Omnibus-Type/Texturina/blob/master/CONTRIBUTORS.txt):** Includes contact information for the project's contributors.
 
+<<<<<<< HEAD
```

Comment:
Merge conflict has been included.

Reply:
Oh dear, sorry, I didn't realize that phantom was still there >.<. I'll solve it.